Cargo.toml

```diff
@@ -7,6 +7,9 @@ version = "1.2.0"
 authors = ["debris <marek.kotewicz@gmail.com>"]
 keywords = ["sha3", "sha-3", "keccak", "crypto", "tiny"]
 
+[dependencies]
+typenum = "1.7.0"
```

Comment:
we should use `no_std` feature of typenum to preserve backwards compatibility

Reply:
https://github.com/paholg/typenum/blob/a9eef9683f7fa8292707a5dad59fc4da18315847/.travis.yml#L23

typenum is `no_std`, the `no_std` feature for compatibility only.